ui/jobs/jobs.js

```diff
@@ -565,6 +578,18 @@ class Bars {
       this.o.healthBar.fg = computeBackgroundColorFrom(this.o.healthBar, 'hp-color');
   }
 
+  _updateProcBoxNotifyState() {
+    if (this.options.NotifyExpiredProcsInCombat >= 0) {
+      const boxes = document.getElementsByClassName('proc-box');
+      for (const box of boxes) {
+        if (this.inCombat)
+          box.classList.add('in-combat');
+        else
+          box.classList.remove('in-combat');
```

Comment:
Thanks for this whole patch! This is a great idea, and I agree with the feedback to not flash things until they've been used once.  My only feedback is this one suggestion to also reset the notify state when you start combat again.  This just extends the "until they've been used once" to "until they've been used once in the current combat."

This would prevent a situation where a proc expires, then going out of combat, and then zoning into a lower level dungeon without ever changing jobs and having it flash at you the whole time.

What do you think?

```suggestion
        if (this.inCombat) {
          box.classList.add('in-combat');
          for (const child of box.children)
            child.classList.remove('expired');
        } else {
          box.classList.remove('in-combat');
        }
```